website/cue/reference/releases/0.24.0.cue

```diff
@@ -33,6 +33,7 @@ releases: "0.24.0": {
 		- A new `opentelemetry` source to receive input from OpenTelemetry collectors and SDKs. Only
 		  logs are supported in this release, but support for metrics and traces are in-flight.
 		  An `opentelemetry` sink will follow.
+		- Support for expiring high cardinality internal metrics through `expire_metrics_secs`.
```

Comment:
This isn't true for 0.24.0 is it? I think we may just want to add a note here that it is actually in 0.24.1.